### PR TITLE
fixed permission and port issue

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
Signed-off-by: Raffaele Spazzoli <raffaele.spazzoli@gmail.com>

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/raffaelespazzoli/volume-expander-operator
```
Comment (sabre1041) [inline config/manager/kustomization.yaml:7]: This should reference `redhat-cop`?

**Review by sabre1041 — COMMENTED** [review]
One question

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (raffaelespazzoli) [reply config/manager/kustomization.yaml]: it is actually irrelevant, but I restored some more sensible values, here and in another point.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**